webpack/assets/javascripts/react_app/redux/actions/toasts/index.js

```diff
@@ -0,0 +1,20 @@
+import { TOASTS_ADD, TOASTS_HIDE, TOASTS_DELETE } from '../../consts';
+import makeUUID from '../../../common/makeUUID';
+
+export const addToast = toast => {
+  return {
+    type: TOASTS_ADD,
+    payload: {
+      key: makeUUID(),
```

Comment:
It's needed for react implementation - removing adding etc. 
timestamp seems kind of hackish, also not sure if server supports it...